schema/deploy/tables/reporting_requirement.sql

```diff
@@ -0,0 +1,50 @@
+-- Deploy cif:tables/reporting_requirement to pg
+
+
+begin;
+create table cif.reporting_requirement
+(
+  id integer primary key generated always as identity,
+  due_date timestamptz,
+  status cif.reporting_requirement_status not null,
+  comments varchar(10000),
+  certified_by varchar(1000),
+  certified_by_professional_designation varchar(1000),
+  project_id integer references cif.project(id) not null,
+  report_type_id integer references cif.report_type not null
```

Comment:
```suggestion
  project_id integer references cif.project(id) not null,
  report_type_id integer references cif.report_type(id) not null
```

specifying the name of the primary key is optional, but we've been doing it, so let's stay consistent